LibraryManagementSystem/Forms/ReturnTab.cs

```diff
@@ -61,8 +61,14 @@ private void btnReturn_Click(object sender, EventArgs e)
                         txtBookID.Clear();
                     }
                     else
-                    { 
-                        cmd = new SqlCommand("UPDATE booksData SET Status = '" + availText + "' WHERE BookID = '" + bookID + "'", con);
+                    {
+                        cmd = new SqlCommand("UPDATE book SET status = '" + availText + "' WHERE book_id = '" + bookID + "'", con);
+                        cmd.ExecuteNonQuery();
+                        cmd = new SqlCommand("UPDATE borrow_data SET member_id = NULL WHERE book_id = '" + bookID + "'", con);
```

Comment:
Like I mentioned in a previous comment, instead of updating borrow_data here why not delete all the values in one SQL query here? <kbd>cmd = new SqlCommand("DELETE * FROM book WHERE book_id = '" + bookID + "'", con);</kbd>

Reply:
This is returning. I don't wanna delete the info. Also, readability and traceability.